redirect.json

```diff
@@ -26,14 +53,129 @@
     }
   },
   {
-    "redirect": "config_flow_start",
-    "description": "allow starting a config flow",
-    "introduced": "2021.3",
+    "redirect": "automations",
+    "description": "show your automations",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "scenes",
+    "description": "show your scenes",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "scripts",
+    "description": "show your scripts",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "helpers",
+    "description": "show your helper entities",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "tags",
+    "description": "show your tags",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "lovelace_dashboards",
+    "description": "show your Lovelace dashboards",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "lovelace_resources",
+    "description": "show your Lovelace resources",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "people",
+    "description": "show your people",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "zones",
+    "description": "show your zones",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "users",
+    "description": "show your users",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "general",
+    "description": "show your general Home Assistant settings",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "server_controls",
+    "description": "show your server controls",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "info",
+    "description": "show your Home Assistant version information",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "logs",
+    "description": "show your Home Assistant logs",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "customize",
+    "description": "allow to customize your entities",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "profile",
+    "description": "show your Home Assistant user's profile",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_states",
+    "description": "show to raw state of your entities",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_services",
+    "description": "show the call services in developer tools",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_template",
+    "description": "show template editor in developer tools",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_events",
+    "description": "show debug tools for events",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "supervisor",
+    "description": "show your Supervisor dashboard",
+    "introduced": "supervisor-2021.02.10"
+  },
+  {
+    "redirect": "supervisor_system",
```

Comment:
Let's split this into `supervisor_info` and `supervisor_logs`. Once we consolidate the logs, it won't be on 1 spot anymore.